discord_bot/opcodes.py

```diff
@@ -0,0 +1,16 @@
+from enum import IntEnum
+
+
+class Opcodes(IntEnum):
```

Comment:
Should probably add a docstring here linking back to the Discord API spec.

Reply:
Done